addon.txt

```diff
@@ -4,8 +4,8 @@
     "version"           "1"
     "up_date"           "00/00/00"
     "author_name"       "FPtje"
-    "author_email"      "wiki.darkrp.com"
-    "author_url"        "wiki.darkrp.com"
+    "author_email"      ""
+    "author_url"        "https://github.com/FPtje/DarkRP"
```

Comment:
```suggestion
    "author_email"      "darkrp.miraheze.org"
    "author_url"        "darkrp.miraheze.org"
```
Aren't these should be like I suggested?

Reply:
I did the same thing [here](https://github.com/FPtje/DarkRP/blob/d0f5a6245ea68590dbc712d0780d057010597ae4/darkrp.txt#L9) for information.

Reply:
Oh... I see. Okay then.